pkgs/tools/misc/aptly/default.nix

```diff
@@ -33,8 +33,8 @@ buildGoPackage {
 
   postInstall = ''
     rm $bin/bin/man
-    mkdir -p $out/etc/bash_completion.d
-    ln -s ${aptlyCompletionSrc}/aptly $out/etc/bash_completion.d
+    mkdir -p $out/share/bash-completion/completions
+    ln -s ${aptlyCompletionSrc}/aptly $out/share/bash-completion/completions
```

Comment:
> I'm sorry for confusion

No problem :grinning:
> please change to $bin/share/bash-completion/completions/

I made the change you requested and I squashed all the commits into one.